Gems/ROS2/Code/Include/ROS2/Lidar/LidarRaycasterBus.h

```diff
@@ -88,6 +88,7 @@ namespace ROS2
     {
         Points = (1 << 0), //!< return 3D point coordinates
         Ranges = (1 << 1), //!< return array of distances
+        Intensity = (1 << 2), //!< return intensity data
```

Comment:
Seems that plural is used here -> Point**s**, Range**s**. Same below, in line 101.